mvp-spec/operator-api.md

```diff
@@ -122,12 +159,49 @@ The following signature verifications are optional:
 
 For details on how to calculate or verify signatures, see [signatures.md](signatures.md).
 
+### Error handling
```

Comment:
line 153 above: "this is the way to prevent" : truncated sentence

Reply:
fixed